tests/unit/amo/components/TestRatingManager.js

```diff
@@ -599,13 +597,91 @@ describe(__filename, () => {
+      const review = { ...fakeReview, score: 4 };
+      const { addon, userId, store } = createStoreWithLatestReview({ review });
+      store.dispatch(beginDeleteAddonReview({ reviewId: review.id }));
+
+      const root = renderInline({ store, addon, userId });
+
+      expect(root.find(UserRating)).toHaveLength(0);
+
+      const managerRating = root.find(AddonReviewManagerRating);
+      expect(managerRating).toHaveLength(1);
+      expect(managerRating).toHaveProp('rating', review.score);
+    });
+
+    it('also configures AddonReviewManagerRating while deletingReview', () => {
+      const { addon, review, userId, store } = createStoreWithLatestReview();
+      store.dispatch(
+        deleteAddonReview({
+          addonId: addon.id,
+          errorHandlerId: 'some-error-handler',
+          reviewId: review.id,
+        }),
+      );
+
+      const root = renderInline({ store, addon, userId });
+
+      expect(root.find(UserRating)).toHaveLength(0);
+      expect(root.find(AddonReviewManagerRating)).toHaveLength(1);
+    });
+
+    it('prompts to delete a review when beginningToDeleteReview', () => {
+      const addon = createInternalAddon({
+        ...fakeAddon,
+        name: 'uBlock Origin',
+      });
+      const review = { ...fakeReview, body: 'This add-on is nice' };
+      const { userId, store } = createStoreWithLatestReview({ addon, review });
+      store.dispatch(beginDeleteAddonReview({ reviewId: review.id }));
+
+      const root = renderInline({ store, addon, userId });
+
+      const prompt = root.find('.RatingManager-legend').html();
+      expect(prompt).toContain('Are you sure you want to delete your review');
+      expect(prompt).toContain(addon.name);
+    });
+
+    it('prompts to delete a rating when beginningToDeleteReview', () => {
+      const addon = createInternalAddon({
+        ...fakeAddon,
+        name: 'uBlock Origin',
+      });
+      const review = { ...fakeReview, body: undefined, score: 4 };
+      const { userId, store } = createStoreWithLatestReview({ addon, review });
+      store.dispatch(beginDeleteAddonReview({ reviewId: review.id }));
+
+      const root = renderInline({ store, addon, userId });
+
+      const prompt = root.find('.RatingManager-legend').html();
+      expect(prompt).toContain('Are you sure you want to delete your rating');
+      expect(prompt).toContain(addon.name);
+    });
+
+    it('still prompts to delete a review while deletingReview', () => {
+      const { addon, review, userId, store } = createStoreWithLatestReview();
+      store.dispatch(
+        deleteAddonReview({
+          addonId: addon.id,
+          errorHandlerId: 'some-error-handler',
+          reviewId: review.id,
+        }),
+      );
+
+      const root = renderInline({ store, addon, userId });
+
+      const prompt = root.find('.RatingManager-legend').html();
+      expect(prompt).toContain('Are you sure you want to delete');
```

Comment:
Sure, I can change it.

> should we do another separate test for rating?

I don't think it's necessary because the existing test that checks for the word `rating` is enough to imply that it will work for both cases.